package.json

```
@@ -116,7 +116,7 @@
 		"semver": "^5.5.0"
 	},
 	"peerDependencies": {
-		"jest": "^18 || ^19 || ^20.0.1 || ^21 || ^22 || ^23 || ^24 || ^25 || ^26"
+		"jest": "^18 || ^19 || ^20.0.1 || ^21 || ^22 || ^23 || ^24 || ^25 || ^26 || ^27"
```

```suggestion
		"jest": "^18 || ^19 || ^20.0.1 || ^21 || ^22 || ^23 || ^24 || ^25 || ^26 || ^27.2.0"
```
might as well start with the latest v27?

---

Sounds good. I'll change it.